collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTest.java

```diff
@@ -166,7 +166,7 @@ private void setupTestSettings() throws IOException {
 
         assertTrue(new File(storagePathProvider.getDirPath(StorageSubdirectory.SETTINGS)).exists() || new File(storagePathProvider.getDirPath(StorageSubdirectory.SETTINGS)).mkdir());
         assertTrue(new File(storagePathProvider.getDirPath(StorageSubdirectory.SETTINGS) + "/collect.settings").createNewFile());
-        assertTrue(new File(storagePathProvider.getDirPath(StorageSubdirectory.ODK) + "/collect.settings").createNewFile());
+        assertTrue(new File(storagePathProvider.getStorageRootDirPath() + "/collect.settings").createNewFile());
```

Comment:
Ooh, yeah, the odk subdirectory was from the StoragePathProvider PR. Sorry I didn't think through that before! Looks great now.